### PR TITLE
[DEV APPROVED] 7097 remove updated year end tools

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1448)

### CODE REVIEW

File: spec/helpers/budget_warning_spec.rb [hunk 1]
```diff
@@ -24,12 +24,10 @@
 
   describe '#whitelisted pages' do
     context 'english page for tools' do
-      ['redundancy-pay-calculator',
-        'workplace-pension-contribution-calculator',
-        'pension-calculator'].each do |tool_name|
-            it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
-               let(:url) { "www.example.com/en/tools/#{tool_name}/" }
-            end
+      ['redundancy-pay-calculator'].each do |tool_name|
```
Comment (misaka) [inline spec/helpers/budget_warning_spec.rb]: Why loop here? Are you expecting to have re-add more calculators? Even if so it'd probably be worth it to remove the looping until then.

File: spec/helpers/budget_warning_spec.rb [hunk 2]
```diff
@@ -41,18 +39,12 @@
...
-      it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
          let(:url) { "www.example.com/en/retirement-income-options/retirement-options" }
       end
     end
 
     context 'welsh pages for tools' do
-       ['cyfrifiannell-tal-diswyddo',
-        'cyfrifiannell-cyfraniadau-pensiwn-gweithle',
-        'cyfrifiannell-pensiwn'].each do |tool_name|
+       ['cyfrifiannell-tal-diswyddo'].each do |tool_name|
```
Comment (misaka) [inline spec/helpers/budget_warning_spec.rb]: Same comment as above re: removing the loop.

File: spec/helpers/budget_warning_spec.rb (hunk 1 above)
Comment (Tr4pSt3R) [inline spec/helpers/budget_warning_spec.rb]: good shout @misaka i will remove that

File: spec/helpers/budget_warning_spec.rb [hunk 3]
```diff
@@ -21,17 +21,8 @@
...
-    context 'english page for tools' do
-      ['redundancy-pay-calculator',
-        'workplace-pension-contribution-calculator',
-        'pension-calculator'].each do |tool_name|
-            it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
-               let(:url) { "www.example.com/en/tools/#{tool_name}/" }
-            end
-      end
-    end
+  it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
```
Comment (misaka) [inline spec/helpers/budget_warning_spec.rb]: You've removed the `describe` and the `context` blocks here. I think those are useful, how about merging it with the `describe` block below (line 28) and adding the same context that used to be here: "english page for tools"?

File: spec/helpers/budget_warning_spec.rb [hunk 4]
```diff
@@ -41,24 +32,12 @@
...
-       ['cyfrifiannell-tal-diswyddo',
-        'cyfrifiannell-cyfraniadau-pensiwn-gweithle',
-        'cyfrifiannell-pensiwn'].each do |tool_name|
-         describe tool_name do
-           it_behaves_like 'displays_banner_warning_for_whitelisted_welsh_tools' do
-             let(:url)      { "www.example.com/cy/tools/#{tool_name}/" }
-           end
-         end
-       end
+    it_behaves_like 'displays_banner_warning_for_whitelisted_welsh_tools' do
```
Comment (misaka) [inline spec/helpers/budget_warning_spec.rb]: Same comment as above.

File: spec/helpers/budget_warning_spec.rb (hunk 3 above)
Comment (Tr4pSt3R) [inline spec/helpers/budget_warning_spec.rb]: on it :)

**Comment by misaka** [comment]
:+1: ... I think that looks right now. :)

**Comment by jiannis** [comment]
If you squash those last two commits and maybe reword the commit messages to get rid of typos then 🚀 🎉

**Comment by jiannis** [comment]
👍

**Comment by Tr4pSt3R** [comment]
Thanks @misaka 
Thanks @jiannis
